patches/build-config-BUILDCONFIG.gn.patch

```diff
@@ -15,7 +15,7 @@ index 9e38ee0e592b64f2457dd48110aceafff95164cd..c09b86450ee9906373c3937f9a1e30cb
    "//build/config/coverage:default_coverage",
    "//build/config/sanitizers:default_sanitizer_flags",
  ]
--
++
```

Comment:
we don't want to add or delete whitespace. It looks like the old patch was incorrectly removing a whitespace line and now this patch adds one